sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -336,6 +336,123 @@ def begin_recognize_business_cards_from_url(self, business_card_url, **kwargs):
                 )
             raise e
 
+    @distributed_trace
+    def begin_recognize_id_documents(self, id_document, **kwargs):
+        # type: (Union[bytes, IO[bytes]], Any) -> LROPoller[List[RecognizedForm]]
+        """Extract field text and semantic values from a given ID document.
+        The input document must be of one of the supported content types - 'application/pdf',
+        'image/jpeg', 'image/png', 'image/tiff' or 'image/bmp'.
+        Alternatively, use 'application/json' type to specify the location (Uri) of the
```

Comment:
You can remove the sentence starting with "Alternatively..." since this is the file stream method. edit: same with async method